test/assets/FinancialTransactionsTestConstants.scala

```diff
@@ -187,4 +188,15 @@ object FinancialTransactionsTestConstants extends ImplicitDateFormatter {
     financialTransactions = Some(Seq(transactionModel(taxYear)))
   )
 
+  val paidBillsViewModel: BillsViewModel = BillsViewModel(
```

Comment:
Could these view models be put somewhere else? This constants object is related to the Financial Transactions API.